pkgs/os-specific/darwin/duti/default.nix

```diff
@@ -10,15 +16,14 @@ stdenv.mkDerivation rec {
     sha256 = "1pg4i6ghpib2gy1sqpml7dbnhr1vbr43fs2pqkd09i4w3nmgpic9";
   };
 
-  nativeBuildInputs = [autoreconfHook];
-  buildInputs = [ApplicationServices];
-  configureFlags = [
-    "--with-macosx-sdk=/homeless-shelter"
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = lib.optional stdenv.hostPlatform.isDarwin apple-sdk_11;
```

Comment:
Are you sure this is necessary? It does not look like it uses any macOS 11 APIs, so `buildInputs` should be able to be omitted entirely.